<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Deposit</class>
 <widget class="QWidget" name="Deposit">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>892</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
background-color: qlineargradient(x1: 0, y1: 0, x2: 0, y2: 1, stop: 0 #5462c3, stop: 1 #ba872c);
}

QLabel {
	background-color: none;
}

QCheckBox{
	background-color: none;
}

QListWidget{
	background-color: none;
}

QPushButton{
	background-color: none;
}

QDateEdit{
	background-color:  rgb(50, 50, 50);
}</string>
  </property>
  <widget class="QLabel" name="total_amount_box">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>840</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QComboBox" name="frequency_payments_combobox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>150</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>19</pointsize>
    </font>
   </property>
   <property name="tabletTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="currentText">
    <string>В конце срока</string>
   </property>
   <item>
    <property name="text">
     <string>В конце срока</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="interest_rate_label">
   <property name="geometry">
    <rect>
     <x>429</x>
     <y>9</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Процентная ставка, % годовых</string>
   </property>
  </widget>
  <widget class="QSpinBox" name="posting_period_box">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>150</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);
border: none;
padding-right: 10px;</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="calculate_push_button">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>740</y>
     <width>171</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font/>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Рассчитать</string>
   </property>
  </widget>
  <widget class="QLabel" name="frequency_payments_label">
   <property name="geometry">
    <rect>
     <x>430</x>
     <y>110</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Периодичность выплат</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount_box">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>840</y>
     <width>221</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="interest_rate_box">
   <property name="geometry">
    <rect>
     <x>421</x>
     <y>50</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QLabel" name="posting_period_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>110</y>
     <width>161</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Срок размещения</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="total_deposit_box">
   <property name="geometry">
    <rect>
     <x>31</x>
     <y>50</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string> ₽</string>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QLabel" name="interes_charges_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>790</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Начисленные проценты</string>
   </property>
  </widget>
  <widget class="QLabel" name="tax_amount_label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>790</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма налога</string>
   </property>
  </widget>
  <widget class="QLabel" name="interes_charges_box">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>840</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);
color: white;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="total_amount_label">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>790</y>
     <width>241</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Сумма на вкладе к концу срока</string>
   </property>
  </widget>
  <widget class="QLabel" name="total_deposit_label">
   <property name="geometry">
    <rect>
     <x>39</x>
     <y>9</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Общая сумма вклада</string>
   </property>
  </widget>
  <widget class="QComboBox" name="posting_period_combobox">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>150</y>
     <width>109</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>6</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);
border:  none;
border-left: 1px solid rgb(100,100,100);</string>
   </property>
   <property name="currentText">
    <string>лет</string>
   </property>
   <property name="maxVisibleItems">
    <number>14</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>лет</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>месяцы</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="tax_rate_box">
   <property name="geometry">
    <rect>
     <x>32</x>
     <y>261</y>
     <width>281</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string> %</string>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QLabel" name="tax_rate_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>220</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Налоговая ставка</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="interest_capitalization_checkbox">
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>260</y>
     <width>261</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>Капитализация процентов</string>
   </property>
  </widget>
  <widget class="QLabel" name="replenishments_label">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>330</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Пополнения</string>
   </property>
  </widget>
  <widget class="QLabel" name="withdrawals_label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>530</y>
     <width>151</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>16</pointsize>
    </font>
   </property>
   <property name="text">
    <string>Частичные снятия</string>
   </property>
  </widget>
  <widget class="QListWidget" name="replenishments_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>440</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="replenishments_combobox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>370</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>6</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);
border:  none;
border-left: 1px solid rgb(100,100,100);</string>
   </property>
   <property name="currentText">
    <string>Разовое</string>
   </property>
   <property name="maxVisibleItems">
    <number>14</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Разовое</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QDoubleSpinBox" name="replenishments_box">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>370</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string> ₽</string>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QPushButton" name="replenishments_push_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>370</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QListWidget" name="withdrawals_list">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>640</y>
     <width>351</width>
     <height>81</height>
    </rect>
   </property>
  </widget>
  <widget class="QComboBox" name="withdrawals_combobox">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>570</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Minimum" vsizetype="Minimum">
     <horstretch>6</horstretch>
     <verstretch>6</verstretch>
    </sizepolicy>
   </property>
   <property name="minimumSize">
    <size>
     <width>0</width>
     <height>0</height>
    </size>
   </property>
   <property name="sizeIncrement">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="baseSize">
    <size>
     <width>0</width>
     <height>7</height>
    </size>
   </property>
   <property name="font">
    <font>
     <pointsize>17</pointsize>
    </font>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);
border:  none;
border-left: 1px solid rgb(100,100,100);</string>
   </property>
   <property name="currentText">
    <string>Разовое</string>
   </property>
   <property name="maxVisibleItems">
    <number>14</number>
   </property>
   <property name="insertPolicy">
    <enum>QComboBox::InsertAtBottom</enum>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <item>
    <property name="text">
     <string>Разовое</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в месяц</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в квартал</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в полгода</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>Раз в год</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="withdrawals_push_button">
   <property name="geometry">
    <rect>
     <x>680</x>
     <y>570</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>+</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="replenishments_month_box">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>370</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Месяц пополнения</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="withdrawals_month_box">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>570</y>
     <width>161</width>
     <height>51</height>
    </rect>
   </property>
   <property name="autoFillBackground">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(50, 50, 50);</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhDigitsOnly</set>
   </property>
   <property name="inputMask">
    <string/>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="placeholderText">
    <string>Месяц пополнения</string>
   </property>
  </widget>
  <widget class="QDoubleSpinBox" name="withdrawals_box">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>570</y>
     <width>231</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>21</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color:  rgb(50, 50, 50);</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
   </property>
   <property name="buttonSymbols">
    <enum>QAbstractSpinBox::NoButtons</enum>
   </property>
   <property name="suffix">
    <string> ₽</string>
   </property>
   <property name="maximum">
    <double>99999999.989999994635582</double>
   </property>
  </widget>
  <widget class="QPushButton" name="replenishments_push_button_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>440</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
  <widget class="QPushButton" name="withdrawals_push_button_2">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>640</y>
     <width>51</width>
     <height>51</height>
    </rect>
   </property>
   <property name="text">
    <string>-</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
